### PR TITLE
[FEATURE]actionType field in update batch operation

### PR DESCRIPTION
**Pull request by iariasleon**
```
                    SUMMARY:
 --------------------------------------------------------------------------------------------------------------------------------------------
|                                        name                                            | passed | failed | skipped |  total  | duration(s) |
 --------------------------------------------------------------------------------------------------------------------------------------------
| components\ngsiv2\batch_operations\batch_op_update\batch_op_update_action_type.feature |      9 |      0 |      15 |      24 |       3.688 |
 --------------------------------------------------------------------------------------------------------------------------------------------
|  Date: Friday, November 11, 2016 10:41:18
 -------------------------------------------------
1 feature passed, 0 failed, 0 skipped
9 scenarios passed, 0 failed, 15 skipped
53 steps passed, 0 failed, 75 skipped, 0 undefined
Took 0m3.688s
```
PR associated: https://github.com/telefonicaid/iotqatools/pull/126
@fgalan @crbrox

### CODE REVIEW

File: test/acceptance/behave/components/ngsiv2/batch_operations/batch_op_update/batch_op_update_action_type.feature [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+      | attributes_name    | temperature |
+      | attributes_value   | 45          |
+      | attributes_type    | kelvin      |
+    When update entities in a single batch operation "UPDATE"
+      | parameter | value     |
+      | options   | keyValues |
+    Then verify that receive a "No Content" http code
+
+  @action_type_delete
+  Scenario:  delete several entities  with batch operations) using NGSIv2 in normalized format
```
Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/batch_operations/batch_op_update/batch_op_update_action_type.feature]: typo: )

Comment (fgalan) [inline test/acceptance/behave/components/ngsiv2/batch_operations/batch_op_update/batch_op_update_action_type.feature]: The same typo seems to be in many places in the file. Please check it globally.

Comment (iariasleon) [inline test/acceptance/behave/components/ngsiv2/batch_operations/batch_op_update/batch_op_update_action_type.feature]: fixed in  	2187773

**Comment by fgalan** [comment]
LGTM

**Pull request merged**